### ISSUE TITLE
Non-druid flight conserves momentum on mid-air dismount.

### ISSUE THREAD

**Issue by Kibir34** [Stormforge-gg/Bugtracker-Fordring#1637]
**Description**: 
Dismounting mid-air as a non-druid leads to momentum being saved, causing the player to go 280% speed for a half second in the air.

**Current behaviour**:
Dismounting mid-air as a non-druid conserves momentum.

**Expected behaviour**: 
Players should be reduced to maximum run speed on dismount.

**Revision**: 
dunno lol .s info doesnt show

**Comment by Vishiswaz** [comment]
To clarify, this should only be when dismounting off a traditional mount, NOT with druid flight forms, which should preserve forward momentum.

**Comment by Cold-Martijn** [comment]
Druid flight form should have momentum, all other mounts should NOT have momentum in WotLK. Confirmed.

**Comment by Cold-Martijn** [comment]
@Aithne99 why you removed the confirmed bug tag? This is an actual bug, and it happens on this server.......

**Comment by Vishiswaz** [comment]
Definitely a bug

**Comment by Cold-Martijn** [comment]
> Definitely a bug

yeah, just noticed Aithne completely deleted the confirmed bug tag lol.

**Comment by Vishiswaz** [comment]
I was agreeing with you
